gui/wxpython/psmap/dialogs.py

```diff
@@ -50,6 +50,7 @@
     from wx import PyValidator as Validator
 
 import grass.script as grass
+from grass.pygrass.utils import findmaps
```

Comment:
Let's not use pygrass, it may be importing ctypes. gs.find_file is safer.

Reply:
All right. Is it a general rule not to use the pygrass module (due ctypes) with wxPython?

Reply:
I never got to answering this... I wouldn't say it's a rule, but I've seen to many incidents, especially on windows, when the entire GUI didn't start because there was something wrong with ctypes. Parts of pygrass do not access C code, but there are not well isolated. So unless you actually need to access the C library in the GUI (like the 3D, digitizer), it's better to avoid it. If you can't avoid it, you need to check for import errors.